src/sqswatcher/sqswatcher.py

```diff
@@ -263,11 +263,16 @@ def _parse_sqs_messages(messages, table):
     return update_events.values()
 
 
-def _process_compute_ready_event(message_attrs, message):
+def _process_compute_ready_event(sqs_config, message_attrs, message):
     instance_id = message_attrs.get("EC2InstanceId")
+    instance_type = message_attrs.get("EC2InstanceType")
+    # Retrieving instances properties for each event because instance types
+    # from instance and CloudFormation could be out-of-sync
+    instance_properties = get_instance_properties(sqs_config.region, sqs_config.proxy_config, instance_type)
```

Comment:
we really don't want to retrieve this file for every new node especially since the update of the compute instance type is a very rare event. We could implement some very simple caching mechanism in get_instance_properties:

```
def get_instance_properties(...):
     if not hasattr(get_instance_properties, "cache"):
          cache = {}
     if instance_type in get_instance_properties.cache:
          return cache[instance_type]

     same as now + update cache before return
```

Cache could also be simply the last item.